dashboard/api.py

```diff
@@ -172,27 +172,31 @@ def get_info_for_course(user, course, user_enrollments, user_certificates):
     # remove the picked run_status from the list
     run_statuses.remove(run_status)
 
-    # an index to sort the runs for a course
-    index = 1
-    if run_status.status in (CourseRunStatus.NOT_ENROLLED, CourseRunStatus.NOT_PASSED):
+    if run_status.status == CourseRunStatus.NOT_ENROLLED:
         next_run = course.get_next_run()
         status = CourseStatus.OFFERED if next_run is not None else CourseStatus.NOT_OFFERED
         course_data['status'] = status
         if next_run is not None:
```

Comment:
It seems like the `next_run is None` case is removed here, is it handled elsewhere? Code coverage was good before so I assume it is but I wanted to double check